### ISSUE TITLE
In scenario 6 is not clear what happens between step 5 and 6.

### ISSUE THREAD

**Issue by phochste** [antleaf/notify-implementation#2]
In scenario 6 there are the following steps (actors in **bold**):

1. An author send a manuscript to a **Journal** 
2. The **Journal** request that a **Repository** ingests the manuscripts (using an AS2 `Offer`)
3. The **Repository** creates a new resource
4. The **Repository** notifies the **Journal** of the new resource (using an AS2 `Announce`)
5. The **Journal** invites reviewers
6. The **Repository** receives reviews for the manuscript (which it links to the original manuscript)
7. The **Repositoy** notifies the **Journal** of the new review resource (using an AS2 `Announce`)
8. The **Journal** endorses the manuscript
9. The **Journal** notifies the **Repository** (using an AS2 `Announce`)
10. The **Repository** assigns a DOI and updates the manuscript metadata

In step 1-4 it is clear that there are two actors: the Journal and a Repository.

In step 5 it is assumed that the Journal invites reviewers that submit reviews *to the same Repository*? Is it the case that the reviewers are part of the same ecosystem as the Repository of steps 1-4? Or, could it be the case that the repository of step 1-4 is different from steps 6-10 and that multiple notifications should be sent to multiple repositories?

I find Scenario 6 one of the most complex ones to get my head around.

**Comment by paulwalk** [comment]
> Is it the case that the reviewers are part of the same ecosystem as the Repository of steps 1-4?

Yes, this is the correct assumption.

Your comment highlights a weakness which I had already suspected in this approach to providing examples. These scenarios were sourced from the project partners and so some of them represent *particular* systems & services. However, we have been trying to represent the scenarios in as generic a way as possible, to encourage their re-use.

For a while now I have been thinking that we might need a third layer of examples, so that we would have:

1. the very basic, generic examples of notifications in the notification patterns (as now)
2. generic scenarios, composed of the patterns in (1)
3. specific scenarios (naming particular systems) composed of the patterns in (1)

I think the reason that scenario 6 feels 'strange' is because it is actually an example of (3) masquerading as an example of (2).

The only reason I haven't immediately acted on this is because it will add more complexity, and may confuse matters further.

Perhaps a better alternative is to keep what we have, but add a bit more context in the general descriptions of the scenarios, explaining where they have come from etc.?

**Comment by phochste** [comment]
Yes in this case a bit of context would be very helpful. I would also like to keep the examples as is because it makes it easier to comment on them and refer to them in documentation. 

This scenario also shows that `Offer` and `Accept` AS2 notifications can be send in both directions from/to Repositories and Journals. This is very enlightening for my implementation viewpoint.

**Comment by paulwalk** [comment]
> Yes in this case a bit of context would be very helpful. I would also like to keep the examples as is because it makes it easier to comment on them and refer to them in documentation.

OK - agreed. I plan to add some descriptions in the next few days.

> This scenario also shows that Offer and Accept AS2 notifications can be send in both directions from/to Repositories and Journals. This is very enlightening for my implementation viewpoint.

Absolutely!
